tests/test_migrations.py

```diff
@@ -41,10 +42,20 @@ def test_migrations_and_tasks(tmp_path: Path) -> None:
         git("commit", "--allow-empty", "-m2")
         git("tag", "v2.0")
     # Copy it in v1
-    run_copy(src_path=str(src), dst_path=dst, vcs_ref="v1.0.0", unsafe=True)
+    run_copy(
+        src_path=str(src),
+        dst_path=dst,
+        vcs_ref="v1.0.0",
+        unsafe=True,
+        skip_tasks=skip_tasks,
+    )
     # Check copy was OK
-    assert (dst / "created-with-tasks.txt").read_text() == "task 1\ntask 2\n"
-    assert not (dst / "delete-in-tasks.txt").exists()
+    if not skip_tasks:
+        assert (dst / "created-with-tasks.txt").read_text() == "task 1\ntask 2\n"
+        assert not (dst / "delete-in-tasks.txt").exists()
+    else:
+        assert not (dst / "created-with-tasks.txt").exists()
+        assert (dst / "delete-in-tasks.txt").exists()
```

Comment:
It's usually more readable to start with non-negated conditions:

```suggestion
    if skip_tasks:
        assert not (dst / "created-with-tasks.txt").exists()
        assert (dst / "delete-in-tasks.txt").exists()
    else:
        assert (dst / "created-with-tasks.txt").read_text() == "task 1\ntask 2\n"
        assert not (dst / "delete-in-tasks.txt").exists()

```

Reply:
I hesitated because depending on projects I contribute to, some prefer
- if non-negated else
- if nominal case else
I chose the 2nd because it was the existing case but I prefer the first one. Submitting the change ASAP